### DISCUSSION TITLE
Uploader，可预览的两个状态是可以上传pdf的，点击选择以后pdf的选项时高亮的，其他状态不可以？

### DISCUSSION THREAD

**Discussion by wwwkj** [youzan/vant#9737] [Q&A] [answered]
https://vant-contrib.gitee.io/vant-weapp/#/uploader

急需解答蟹蟹

**Comment by chenjiahan** [comment, marked as answer]
其他状态指的是啥？

**Reply by wwwkj** [reply]
举例：基础用法里的上传框pdf是灰掉的不可选的，但是下面的文件预览的上传框pdf是亮起的可选的。

**Reply by wwwkj** [reply]
我知道怎么回事了谢谢 。accept="all" 就可以了。但是这个限制没有写进对应demo的代码里，导致我被误导了。
